packages/babel-plugin-proposal-object-rest-spread/test/fixtures/object-spread/no-object-assign-exec/exec.js

```diff
@@ -22,3 +22,11 @@ const obj2Spread = { ...obj2 };
 
 assert.deepEqual(obj, objSpread);
 assert.deepEqual(obj2, obj2Spread);
+
+// const obj3Spread = { ...{ get foo () { return 'bar' } } };
+// assert.equal(Object.getOwnPropertyDescriptor(obj3Spread, 'foo').value, 'bar')
```

Comment:
This one is commented out because it obviously fails until the above will be fixed…